### DISCUSSION TITLE
Can I simulate CSNN (Convolutional Spiking Neural Network) on aihwkit ?

### DISCUSSION THREAD

**Discussion by sauravtii** [IBM/aihwkit#430] [General]
Hello!
I am new to aihwkit and wanted to ask whether I can simulate Convolutional Spiking Neural Network (CSNN) using aihwkit ?
If yes, can anyone please share the necessary resources from where I can get started ?
I have a custom CNN and want to convert that to SNN and simulate or run it on aihwkit in order to get the performance metrics (accuracy, latency, energy, etc).

Kind regards,
Saurav

**Comment by maljoras** [comment]
Hi @sauravtii ,
many thanks for the question!  

No, we do not support spiking neuromorphic hardware. Just resistive crossbar arrays.

**Comment by sauravtii** [comment]
Thank you for your response @maljoras. Is it possible to run or simulate CSNN (Convolutional Spiking Neural Network) using resistive crossbar arrays in order to get the performance metrics ?
Or is there any possible way to run SNN using aihwkit ?

**Comment by maljoras** [comment]
No, spiking DNNs are not supported.
